src/routes/auth.js

```diff
@@ -50,4 +50,47 @@ router.post('/signup', async (request, response, next) => {
   }
 });
 
+router.post('/signin', async (request, response, next) => {
+  try {
+    const userEmail = request.body.email;
+    const userPassword = request.body.password;
+
+    if (!userEmail || !validator.isEmail(userEmail)) {
+      throw new RequestError('Must provide a valid email');
+    }
+    if (!userPassword || !userPassword.trim()) {
+      throw new RequestError('Must provide a valid password');
+    }
+
+    const user = await Prisma.user.findUnique({
+      where: {
+        email: userEmail,
+      },
+    });
+    const passwordMatch = await bcrypt.compare(userPassword, user.password);
+    if (!user && !passwordMatch) {
+      throw new RequestError('Invalid email or password');
+    }
+
+    const accessToken = await tokenService.createAccessToken(user);
+    const refreshToken = await tokenService.getSignedRefreshToken({
```

Comment:
In our `Schema.prisma` file, the user model has a RefreshTokens attribute as a list of RefreshTokens. does the token created on this line get added to the list oh RefreshTokens in the specific user?

Reply:
Yes it does